routes/users.go

```diff
@@ -8,8 +8,11 @@ import (
 )
 
 func UserRoutes(rg *gin.RouterGroup, db *bun.DB) {
+
     users := rg.Group("/users")
+    user := rg.Group("/user")
     users.Use(middleware.AuthMiddleware())
```

Comment:
why there is 2 route /users and /user

Reply:
See API endpoints i shared in the discord channel